test/test_pipeline/components/feature_preprocessing/test_kernel_pca.py

```diff
@@ -36,11 +37,11 @@ def test_default_configuration_classify(self):
             X_test_trans = preprocessor.transform(X_test)
```

Comment:
@mfeurer : pushed this fix for the kernel PCA test. Although there were two missing random_state assignments, I think the main problem was caused by this line. It was having numerical issues and returning non-deterministic `X_test_trans`. This problem does not occur with other datasets (e.g. iris) or if we use 'digits' with a larger training dataset. So I increased it to 1000 samples.

Reply:
This appears to be an open issue: https://github.com/scikit-learn/scikit-learn/issues/5970